From: Michael Tokarev <mjt@tls.msk.ru>
Subject: Remove bashism from Makefile

Makefile uses [ x == y ] construct which does not work
with POSIX shell.  Since this is just testing a flag,
replace it with string comparison (=) operator instead.

Signed-off-By: Michael Tokarev <mjt@tls.msk.ru>

diff --git a/Makefile b/Makefile
index e8da3a5..c60cc2c 100644
--- a/Makefile
+++ b/Makefile
@@ -156,7 +156,7 @@ all : check_rundir mdadm mdmon
 man : mdadm.man md.man mdadm.conf.man mdmon.man raid6check.man
 
 check_rundir:
-	@if [ ! -d "$(dir $(RUN_DIR))" -a  "$(CHECK_RUN_DIR)" == 1 ]; then \
+	@if [ ! -d "$(dir $(RUN_DIR))" -a  "$(CHECK_RUN_DIR)" = 1 ]; then \
 		echo "***** Parent of $(RUN_DIR) does not exist.  Maybe set different RUN_DIR="; \
 		echo "*****  e.g. make RUN_DIR=/dev/.mdadm" ; \
 		echo "***** or set CHECK_RUN_DIR=0"; exit 1; \
